testing/testlog/example_test.go

```diff
@@ -0,0 +1,28 @@
+package testlog_test
+
+import (
+	"testing"
+
+	"github.com/heroku/x/testing/testlog"
+)
+
+func Example() {
+	var t *testing.T // use t provided by test function
```

Comment:
This is a bit awkward, since there's no way to get a proper T here in the same way an end-user would. One option would be to omit these testable examples and put them into plain docs (which is how the [testing package](https://golang.org/pkg/testing/) shows examples).